pkg/services/validationService.go

```diff
@@ -192,11 +191,41 @@ func Validate(admissionReviewReq *admission.AdmissionReview, warningMessages *[]
 	}
 
 	clusterRequestMetadata := getClusterRequestMetadata(cliEvaluationId, token, false, allowed, resourceKind, resourceName, managers, clusterK8sVersion, policy.Name, namespace, server.ConfigMapScanningFilters)
-	go cliClient.SendRequestMetadata(clusterRequestMetadata)
-
+	saveRequestMetadataLogInAggregator(clusterRequestMetadata)
 	return ParseEvaluationResponseIntoAdmissionReview(admissionReviewReq.Request.UID, allowed, msg, *warningMessages), false
 }
 
+type ClusterRequestMetadataAggregator = map[string]*cliClient.ClusterRequestMetadata
+
+var clusterRequestMetadataAggregator = make(ClusterRequestMetadataAggregator)
+
+func saveRequestMetadataLogInAggregator(clusterRequestMetadata *cliClient.ClusterRequestMetadata) {
+	hashInBytes, err := json.Marshal(clusterRequestMetadata)
+	if err != nil {
+		panic(err)
```

Comment:
I think that panic is for cases which the err is important, here is not important for the user / business